client/assets/js/admin-controller.js

```diff
@@ -1,8 +1,12 @@
-angular.module('portal').controller('adminController', function ($scope, $http, $location) {
+angular.module('portal').controller('adminController', function ($route, $scope, $http, $location) {
 
     // page title
     $scope.title = 'admin';
 
+    // adding new admin and projx team member 
+    $scope.changeAdminEmail = '';
```

Comment:
i did a quick search and it seems `$scope` is only necessary if you need to share variables between the html and the js. i think a lot of these variables could just be declared normally (eg. `let changeAdminEmail = '';`) if they aren't used in the html